src/test/java/com/snowflake/kafka/connector/records/RecordContentTest.java

```diff
@@ -125,103 +130,98 @@ public void test() throws IOException {
             "{\"int8\":12,\"int16\":12,\"int32\":12,\"int64\":12,\"float32\":12.2,\"float64\":12.2,\"boolean\":true,\"string\":\"foo\",\"bytes\":\"Zm9v\",\"array\":[\"a\",\"b\",\"c\"],\"map\":{\"field\":1},\"mapNonStringKeys\":[[1,1]]}");
   }
 
-  @Test(expected = SnowflakeKafkaConnectorException.class)
-  public void testEmptyValueDisabledTombstone() {
+  @ParameterizedTest
+  @MethodSource("disabledTombstoneSource")
+  public void recordService_getProcessedRecordForSnowpipe_whenDisabledTombstone_throwException(
+      String testCase, SinkRecord record) {
```

Comment:
Let's use `Named` arguments instead of passing a test case name. See [a stackoverflow thread](https://stackoverflow.com/questions/57892989/generating-display-names-for-parameterizedtest-in-junit-5).

Reply:
Thanks, good catch! Syntax for multiple parameters looks weird but even though it is cleaner.